sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -583,7 +580,7 @@ async def get_topic_runtime_properties(
         if not entry.content:
             raise ResourceNotFoundError("Topic {} does not exist".format(topic_name))
         topic_description = TopicRuntimeProperties._from_internal_entity(
-            topic_name, entry.content.topic_description
+            topic_name, cast(TopicDescription, entry.content.topic_description)
```

Comment:
Just curious, why does this have to be cast here does entry.content.topic_description have no type?

Reply:
When it's returned, it's type `Optional[TopicDescription]`, but the _from_internal_entity method only takes type `TopicDescription` and not `None`, so had to cast.